index.html

```diff
@@ -19,27 +19,48 @@
  
     <header>
       <!-- MOBILE-&-TABLET-VERSION -->
-      <a href="">Welcome</a>
+      <a class="welcome" href="">Welcome</a>
         <nav class="h-mobile">
-           <button id="bmenu" type="button"><span class="material-symbols-outlined"> menu </span></button>
+           <button id="bmenu" class="show" type="button">
+            <span class="line1__bars-menu"></span>
+            <span class="line2__bars-menu"></span>
+            <span class="line3__bars-menu"></span>
+          </button>
         </nav>
-
+ 
         <!-- PC-VERSION -->
         <nav class="h-desktop" >
           <ul>
             <li>
-              <a href="#">Portfolio</a>
+              <a href="#main-container">Portfolio</a>
             </li>
             <li>
-              <a href="#">About</a>
+              <a href="#about-me">About</a>
             </li>
             <li>
-              <a href="#">contact</a>
+              <a href="#footer">contact</a>
             </li>
           </ul>
         </nav>
     </header>
-
+    <div class="cross">
+    <button class="xmenu" type="button">
+      <span><img src="/img/cross.svg" alt=""></span>
+    </button>
+  </div>
+    <div class="list_bar-menu">
+      <ul>
+        <li>
+          <a class="a0" href="#main-container">Portfolio</a>
+        </li>
+        <li>
+          <a class="a1" href="#about-me">About</a>
+        </li>
+        <li>
+          <a class="a2" href="#footer">contact</a>
+        </li>
+      </ul>
+    </div>
```

Comment:
Let's stick to the provided design. The `mobile menu` should have to cover the full screen and it shouldn't have to be scrollable. The only way the user can see the other content of the page is by either choosing one of the options or closing the `mobile menu`. Kindly fix that.

https://user-images.githubusercontent.com/34578631/174819273-2f1973bd-6b51-4927-8879-5192b6e26668.mov